apps/journeys/src/components/AiChat/AiChat.tsx

```diff
@@ -182,6 +184,15 @@ export function AiChat({ open }: AiChatProps) {
           <ConversationScrollButton />
         </Conversation>
         <Suggestions>
+          {suggestionsLoading && (
+            <div className="flex items-center gap-2 px-4 py-2 text-muted-foreground">
+              <Loader className="size-4 animate-spin" />
+              <span>Loading suggestions, please hold...</span>
```

Comment:
Wrap with t function for translating text.

```suggestion
              <span>{t('Loading suggestions, please hold...')}</span>
```